src/BioFSharp/PWM.fs

```diff
@@ -0,0 +1,290 @@
+        let getBPVofBFV (caArray:FrequencyVector.FrequencyVector) =
+            caArray.Array
+            |> Array.map (fun item -> float item)
+            |> fun item -> new ProbabilityVector(item)
+
+        /// Create normalized ProbabilityVector based on FrequencyVector.
+        let getNormalizedProbabilityVector (usedBioSet:#IBioItem[]) (pseudoCount:float) (backGroundFrequencyVector:FrequencyVector.FrequencyVector) =
+            let backGroundProbabilityVector = getBPVofBFV backGroundFrequencyVector
+            let sum = (float (Array.sum backGroundFrequencyVector.Array)) + ((float usedBioSet.Length) * pseudoCount)
+            for item in usedBioSet do
+                backGroundProbabilityVector.[item] <- (backGroundProbabilityVector.[item] + pseudoCount)/sum
+            backGroundProbabilityVector
+
+        /// Calculate the score of the given sequence based on the Probabilities of the ProbabilityVector.
+        let calculateBackGroundSegmentScore (bpVector:ProbabilityVector) (bioItems:BioArray.BioArray<#IBioItem>) =
+            Array.fold (fun (value:float) (bios:#IBioItem) -> value * (bpVector.[bios])) 1. bioItems
+
+    /// Contain tools to create and manipulate PositionFrequencyMatrix.
+    module PositionFrequencyMatrix =
+
+        /// Matrix with fixed positions for nucleotides and amino acids with default value of 0.
+        type PositionFrequencyMatrix internal(matrix:int [,]) =
+
+            inherit BaseMatrix<IBioItem, int>(matrix)
+
+            new (rowLength:int) = 
+                let arr:'value [,] = Array2D.zeroCreate 49 rowLength 
+                new PositionFrequencyMatrix(arr)
+
+        /// Increase counter of PositionFrequencyMatrix at fixed position by 1.
+        let addInPlaceAtPos (pos:int) (bioItem:'a when 'a :> IBioItem) (positionFrequencyMatrix:PositionFrequencyMatrix) = 
+            positionFrequencyMatrix.[bioItem, pos] <- positionFrequencyMatrix.[bioItem, pos] + 1
+            positionFrequencyMatrix
+
+        /// Increase counter of PositionFrequencyMatrix at fixed position by n.
+        let addInPlaceSpecifiedAmountAtPos (pos:int) (bioItem:'a when 'a :> IBioItem) (positionFrequencyMatrix:PositionFrequencyMatrix) n = 
+            positionFrequencyMatrix.[bioItem, pos] <- positionFrequencyMatrix.[bioItem, pos] + n
+            positionFrequencyMatrix
+
+        /// Create PositionFrequencyMatrix based on BioArray.
+        let getPositionFrequencyMatrix (source:BioArray.BioArray<#IBioItem>) =
+            let positionFrequencyMatrix = new PositionFrequencyMatrix(source.Length)
+            source
+            |> Array.fold (fun (row, cm) column -> row + 1, addInPlaceAtPos row column cm) (0, positionFrequencyMatrix) |> ignore
+            positionFrequencyMatrix
+
+        /// Create new PositionFrequencyMatrix based on the sum of the positions of an array of Countmatrices.
+        let fusePositionFrequencyMatrices (motifLength:int) (countMatrices:PositionFrequencyMatrix[]) =
```

Comment:
The function is already in the PositionFrequencyMatrix module. You can name this function `fuse`